plugins/store-utils.js

```diff
@@ -0,0 +1,88 @@
+export const saveToLocalStorage = ( key, values ) => {
+  try {
+    let existing = localStorage.getItem( key );
+    existing = JSON.parse( existing ) || {};
+    const data = JSON.stringify( { ...existing, ...values } );
+    if ( data ) localStorage.setItem( key, data );
+  } catch ( error ) {
+    console.error( 'Failed to save to localStorage', error );
+  }
+};
```

Comment:
Might make more sense to move this to `assets/js` instead of it being a plugin since it doesn't require anything from nuxt (I think). But it can be moved later.

Reply:
(referring to entire file, not just that function)